src/components/application_manager/test/commands/hmi/vr_is_ready_response_test.cc

```diff
@@ -73,7 +73,8 @@ class VRIsReadyResponseTest : public CommandsTest<CommandsTestMocks::kIsNice> {
 };
 
 TEST_F(VRIsReadyResponseTest, RUN_SUCCESS) {
-  MessageSharedPtr command_msg(CreateMessage(smart_objects::SmartType_Map));
+  MessageSharedPtr command_msg(
+      CreateMessage(NsSmartDeviceLink::NsSmartObjects::SmartType_Map));
```

Comment:
@wolfylambova22 Please create an alias for NsSmartDeviceLink::NsSmartObjects as smart_objects and use it. Like it was before.